internal/versions/versions.go

```diff
@@ -249,64 +310,51 @@ func (h *handler) AddReleaseImage(releaseImageUrl, pullSecret, ocpReleaseVersion
 		}
 	}
 
-	// Get minor version
-	ocpVersionKey, err := h.getKey(ocpReleaseVersion)
-	if err != nil {
-		return nil, err
-	}
-
 	// Ensure a relevant OsImage exists
-	osImage, err := h.GetOsImage(ocpVersionKey, cpuArchitecture)
+	osImage, err := h.GetOsImage(ocpReleaseVersion, cpuArchitecture)
 	if err != nil || osImage.URL == nil {
-		return nil, errors.Errorf("No OS images are available for version: %s", ocpVersionKey)
+		return nil, errors.Errorf("No OS images are available for version: %s", ocpReleaseVersion)
 	}
 
 	// Fetch ReleaseImage if exists
-	releaseImage, err := h.GetReleaseImage(ocpVersionKey, cpuArchitecture)
-	if err != nil {
+	releaseImage := funk.Find(h.releaseImages, func(releaseImage *models.ReleaseImage) bool {
```

Comment:
This is probably worth a comment.
Unless you've spent some time looking at this code it's not clear why we can't continue to use `GetReleaseImage` here.

(It's because we don't want the fallback logic in this case, right?)